account_invoice_block_payment/models/account_move.py

```diff
@@ -0,0 +1,23 @@
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    blocked = fields.Boolean(
```

Comment:
"blocked" is too generic here, I recommend using a more descriptive name. 

```suggestion
    payment_blocked = fields.Boolean(
```

TODO: rename everywhere